types/index.d.ts

```diff
@@ -236,21 +241,25 @@ export type ConcreteBatch = {
   query: any;
   text: string | null;
   operationKind: string;
-};
-export type CacheConfig = {
+}
+export interface CacheConfig {
   force?: boolean;
   poll?: number;
   // rerunParamExperimental?: any;
-};
+}
 
 export type Uploadable = File | Blob;
-export type UploadableMap = { [key: string]: Uploadable };
+export interface UploadableMap {
+  [key: string]: Uploadable;
+}
 
 export type RelayObservable<T> = Promise<T>;
```

Comment:
Could you fix this type too, it is wrong. https://github.com/relay-tools/react-relay-network-modern/blob/bf0b16f7a008bdaf72e97a92eeddf9ee4d2d7a38/src/definition.js#L112-L114